### PR TITLE
[FSTORE-426] errors out of date info in the data validation documentation 3.0

### PR DESCRIPTION
**Pull request by vatj**
Cherry pick fix to documentation from hackathon feedback

### CODE REVIEW

File: docs/user_guides/fs/feature_group/data_validation.md [hunk 1]
```diff
@@ -6,25 +6,29 @@ Clean, high quality feature data is of paramount importance to being able to tra
 
 ## UI
 
-### Step 1: Create or Edit a Feature Group
+### Create a Feature Group (Pre-requisite)
 
-You can attach at most one expectation suite to a Feature Group. It can be done on creation or at a later point in time. Data validation is an optional step and is not required to write to a Feature Group. You can find out more information about creating a Feature Group [here](create.md).
+In the UI, one must create a Feature Group first before attaching an Expectation Suite. You can find out more information about creating a Feature Group [here](create.md). You can attach at most one expectation suite to a Feature Group. Data validation is an optional step and is not required to write to a Feature Group.
```
Comment (jimdowling) [inline docs/user_guides/fs/feature_group/data_validation.md:11]: We don't use the "royal we" in docs. It's american style "active voice"

In the UI, you must first create a Feature Group before you can attach an Expectation Suite to it.

File: docs/user_guides/fs/feature_group/data_validation.md [hunk 2]
```diff
@@ -6,25 +6,29 @@ Clean, high quality feature data is of paramount importance to being able to tra
...
+### Step 1: Find and Edit Feature Group
+
+Click on the Feature Group section in the navigation menu. Find your Feature Group in the list and click on its name to access the Feature Group page. Select `edit` in the top right corner or scroll to the Expectations section and click on `Edit Expectation Suite`.
 
 ### Step 2: Edit General Expectation Suite Settings
 
 Scroll to the Expectation Suite section. You can pick a name for your suite as well as two general options.
 
-- enabled checkbox controls whether validation will be run automatically before writing a DataFrame to a Feature Group. Note that validation is executed by the client.
+- Checkbox enabled. It controls whether the Expectation Suite will be used to validate a Dataframe automatically upon insertion into a Feature Group. Note that validation is executed by the client. Disabling validation allows to skip the validation step without deleting the Expectation Suite.
```
Comment (jimdowling) [inline docs/user_guides/fs/feature_group/data_validation.md:21]: It controls whether -> This controls whether

Classic french speaker mistake - "this allows to ..."
Disabling validation allows you to skip

(active voice)

File: docs/user_guides/fs/feature_group/data_validation.md [hunk 3]
```diff
@@ -6,25 +6,29 @@ Clean, high quality feature data is of paramount importance to being able to tra
...
 
 ### Step 3: Add new expectations
 
-By clicking on `Add another expectation` one can choose an expectation type from a dropdown menu. Currently, only the built-in expectations from the Great Expectations framework are supported. For user-defined expectations, please use the Rest API or python client. All default kwargs associated to the selected expectation type are populated as a json below the dropdown menu. Edit the json and click the tick button to save the change locally.
+By clicking on `Add expectation` one can choose an expectation type from a searchable dropdown menu. Currently, only the built-in expectations from the Great Expectations framework are supported. For user-defined expectations, please use the Rest API or python client.
```
Comment (jimdowling) [inline docs/user_guides/fs/feature_group/data_validation.md:26]: searchable dropdown menu -> navigable dropdown menu 
(you don't search it, right?)

File: docs/user_guides/fs/feature_group/data_validation.md [hunk 4]
```diff
@@ -6,25 +6,29 @@ Clean, high quality feature data is of paramount importance to being able to tra
...
+
+All default kwargs associated to the selected expectation type are populated as a json below the dropdown menu. Edit the arguments in the json to configure the Expectation. In particular, arguments such as `column`, `columnA`, `columnB`, `column_set` and `column_list` require valid feature name(s). Click the tick button to save the expectation configuration and append it to the Expectation Suite locally.
```
Comment (jimdowling) [inline docs/user_guides/fs/feature_group/data_validation.md:28]: is the "tick button" a button or is it "Click the checkbox"?

**Review by jimdowling — CHANGES_REQUESTED** [review]
Few small fixes

File: docs/user_guides/fs/feature_group/data_validation.md (hunk 3 above)
Comment (vatj) [reply docs/user_guides/fs/feature_group/data_validation.md:26]: Actually I have added that function in hopsworks-front dev following a request from the hackathon and I am backporting it to 3.0 as soon as I am finished editing this PR

**Review by jimdowling — APPROVED** [review]

**Pull request merged**